.gitignore

```diff
@@ -142,4 +142,11 @@ build.out
 # Azure Project
 csx/
 *.GhostDoc.xml
-pingme.txt
\ No newline at end of file
+pingme.txt
+
+# Build tools that get updated
```

Comment:
@dsgouda can we batch these files with files that are mentioned on line 17-22
Or better create a new group after line 25 and put all these files together.